jetstream.go

```diff
@@ -108,7 +112,9 @@ type jsOpts struct {
 	timeout time.Duration
 	ctx     context.Context
 
-	ack jsAcKOpts
+	ack        jsAcKOpts
```

Comment:
`jsAcKOpts`? There's a capital K and also stutter with 'js'

Reply:
typo, will fix - am happy with the stutter though.